### DISCUSSION TITLE
google authentication is not working

### DISCUSSION THREAD

**Discussion by 1mfaasj** [outline/outline#2404] [Self Hosting]
Hi
It seems that google authentication is not working. I have followed all instructions and am very familiar with docker and also use google authentication for other purposes but can't get this to work.
The button in Outline to log in with Google does not appear, whatever I try. Slack and email work fine.

Is there an option to use Outline without authentication? I use my own authentication with traefik (reversed proxy) so my apps (like Outline) don't have to take care of that.

thanks in advance

what i did:
Created new credentails at https://console.cloud.google.com/apis/credentials, there I got a clientID and secret which I copied into my docker compose file. (at: google _client_id and google_client_secret)
I also filled in the Authorized redirect URI. according to the format described in your .env.

**Comment by lgg42** [comment]
Hey, I'm trying to set it up also.. and just before you hit it...
![image](https://user-images.githubusercontent.com/17387387/128545732-d93a0a21-3a70-4b97-94a2-2a88cf133ec4.png)
For Google Login one must have a Gsuite Workspace.

**Reply by manouazadi** [reply]
Same problem here.
It would be great when there would be an authentication for normal gmail users.
Or a working email authentication.
